gymnasium_robotics/envs/multiagent_mujoco/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""See the [Doc Page](https://robotics.farama.org/envs/fetch/MaMuJoCo)."""
+
+import multiagent_mujoco.mamujoco_v0  # noqa: F401
```

Comment:
Why are you using `noqa: F401`

Reply:
Because this file is not ignored by `flake8`

Reply:
https://github.com/Farama-Foundation/Gymnasium/blob/d71a13588266256a4c900b5e0d72d10785816c3a/.pre-commit-config.yaml#L31
Use the `--per-file-ignores=*/__init__.py:F401`